sdk/notificationhubs/notification-hubs/samples-dev/sendTagsList.ts

```diff
@@ -22,14 +23,14 @@ import {
   NotificationHubsClientContext,
   createClientContext,
 } from "@azure/notification-hubs/client";
-import { SendOperationOptions } from "@azure/notification-hubs/models/options";
+import { SendNotificationOptions } from "../src/models/options.js";
```

Comment:
should be importing from `@azure/notification-hubs`

Reply:
Fixed